commands/core/drupal/update.inc

```diff
@@ -196,6 +205,15 @@ function drush_update_batch() {
     }
   }
 
+  // Apply post update hooks.
+  $post_updates = \Drupal::service('update.post_update_registry')->getPendingUpdateFunctions();
+  if ($post_updates) {
+    $operations[] = ['drupal_flush_all_caches', []];
```

Comment:
Yeah, the patch that just landed calls that function between hook_update_N implementations and the new hook:

```php
    if ($post_updates) {
      // Now we rebuild all caches and after that execute the hook_post_update()
      // functions.
      $operations[] = ['drupal_flush_all_caches', []];
      foreach ($post_updates as $function) {
        $operations[] = ['update_invoke_post_update', [$function]];
      }
    }
```
 I will re-work to use `drush_drupal_cache_clear_all()`.